.github/workflows/build.yml

```diff
@@ -161,6 +215,47 @@ jobs:
       - name: Run integration tests
         run: |
           pipenv run pytest sast_integration_tests
+
+  sast-integration-tests-old-python:
+    strategy:
+      fail-fast: true
+      matrix:
+        python: ["3.8"]
```

Comment:
add versions 3.9-11
(let's define the old-python to be the same across old integration tests)